picky-asn1/src/wrapper.rs

```diff
@@ -387,6 +387,16 @@ impl IntegerAsn1 {
     }
 }
 
+#[cfg(feature = "zeroize")]
+use zeroize::Zeroize;
```

Comment:
I would prefer either putting the `use` statement at the top, or do `impl zeroize::Zeroize …` bellow

Reply:
You are right, my bad.